spec/appliance_console/database_replication_spec.rb

```diff
@@ -115,4 +115,33 @@
       expect(subject.generate_cluster_name).to be_falsey
     end
   end
+
+  context "#write_pgpass_file" do
+    let(:pgpass_path) { Tempfile.new("pgpass").path }
+
+    before do
+      stub_const("#{described_class}::PGPASS_FILE", pgpass_path)
```

Comment:
Okay, I changed how we test the repmgr.conf file, but I kind of like testing this one as it is because it allows us to test the permission changes in a nice way.

I would rather lay a file down and test the permissions with a stat than stub those calls.